tests/foreman/cli/test_subscription.py

```diff
@@ -276,3 +278,48 @@ def test_positive_auto_attach_disabled_golden_ticket(
     with pytest.raises(CLIReturnCodeError) as context:
         target_sat.cli.Host.subscription_auto_attach({'host-id': host_id})
     assert "This host's organization is in Simple Content Access mode" in str(context.value)
+
+
+def test_negative_check_katello_reimport(target_sat, function_org):
+    """Verify katello:reimport trace should not fail with an TypeError
+
+    :id: b7508a1c-7798-4649-83a3-cf94c7409c96
+
+    :steps:
+        1. Import expired manifest & refresh
+        2. Delete expired manifest
+        3. Re-import new valid manifest & refresh
+
+    :expectedresults: There should not be an error after reimport manifest
+
+    :customerscenario: true
+
+    :BZ: 2225534, 2253621
+
+    :CaseImportance: High
+    """
+    remote_path = f'/tmp/{EXPIRED_MANIFEST}'
+    target_sat.put(DataFile.EXPIRED_MANIFEST_FILE, remote_path)
+    # Import expired manifest & refresh
+    target_sat.cli.Subscription.upload({'organization-id': function_org.id, 'file': remote_path})
+    with pytest.raises(CLIReturnCodeError):
+        assert (
+            "Manifest refresh failed:\n  The manifest doesn\'t exist on console.redhat.com."
```

Comment:
```suggestion
  "Manifest refresh failed" in target_sat.cli.Subscription.refresh_manifest({'organization-id': function_org.id})
), f"Optional assertion failed message w/ details"
```
For this assertion, would it be best to assert that "Manifest refresh failed" is in the output? I suggest because i find asserting longer and specific error messages will lead to failure if a small potion of the message has changed down the result, though the result itself remains the same.

Reply:
Keeping `Manifest refresh failed` in assertion would better way, because 
- the rest of error message (`The manifest doesn\'t exist on console.redhat.com. \nPlease create and import a new manifest.`) has different reason to appear in logs